test/DotNetIsolatedInvalidApp/DotNetIsolatedInvalidApp.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This app itself is not invalid. All we care about is using an invalid zip package. Can't we use an existing test app?